### PR TITLE
Added Blog Post on Upcoming Discourse Migration

### PR DESCRIPTION
**Pull request by eeyun**
Signed-off-by: eeyun <ihenry@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by eeyun** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/309115718).

File: www/source/blog/media/2017-11-28-Discourse-Migration.html.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Habitat Discourse Migration"
+date: 2017-11-28
+author: Ian Henry
+tags: forums, discourse
+category: community
+classes: body-article
+---
+
+Hello inHabitants!
```
Comment (mwrock) [inline www/source/blog/media/2017-11-28-Discourse-Migration.html.md]: HA! `inHabitants` :)

**Comment by tashimi** [comment]
My one suggestion is that this needs to include how to still get notified of release notes. Are we going to publish them on the Habitat blog and the Habitat Chef Discourse?

File: www/source/blog/media/2017-11-28-Discourse-Migration.html.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+In our perpetual effort to find better ways to keep our community connected, informed, and cared for we've made the decision to retire the [Habitat Discourse Site](https://forums.habitat.sh) and migrate to something we hope will be more useful. We've been discussing the forums and their efficacy for a while but we've finally come to a conclusion. Looking at metrics from all of our communication tools (github, slack, discourse, and the habitat website) we've found that our community has just not adopted the forum/mailinglist format as a desired communication mechanism.
```
Comment (mwrock) [inline www/source/blog/media/2017-11-28-Discourse-Migration.html.md]: `mailinglist` -> `mailing list`

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/309115718).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/blog/2017-11-28-Discourse-Migration.html.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+## What you need to know
+
+When the migration process begins we will redirect any current hyperlinks directly to the new Discourse instance and update our various integrations to leverage the new account.
+
+#### Migration Dates
+**start date:** Monday, December 4th and will be completed on  
+**end date:** Monday, December 11th.  
+
+
+#### Release Announcements
```
Comment (tashimi) [inline www/source/blog/2017-11-28-Discourse-Migration.html.md]: Perfect!

**Pull request merged**